enhancements/bare-metal-fulfillment/README.md

```diff
@@ -0,0 +1,360 @@
+---
+title: bare-metal-fulfillment
+authors:
+  - Tzu-Mainn Chen
+creation-date: 2025-09-08
+last-updated: 2025-09-15
+tracking-link:
+  - None
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Bare Metal Fulfillment
+
+## Summary
+
+One key component of Virtual Data Center (VDC) fulfillment is the capability to perform bare metal fulfillment. Bare metal
+fulfillment refers to the process through which a tenant acquires bare metal hosts and configures their networking.
+Afterwards, the tenant will be able to perform selected operations on the host: inventory information, power control, and
+serial console access. Any further operations - for example, image-based provisioning - are outside the scope of the O-SAC
+solution and must be integrated by the cloud provider.
+
+We believe this feature set is generic enough to support a wide range of initial bare metal cluster use cases, including:
+
+* support for an environment where developers have to flexibility to install their own operating systems and software
+* support for alternative clusters such as SLURM
+* support for bare metal configuration during O-SAC cluster fulfillment
+
+In this context, a “tenant” is an organization with admin rights to the requested bare metal resources; at the MOC, a tenant
+would be led by a ColdFront PI or manager.
```

Comment:
I'm not sure this information is useful. Here we're only trying to define "what is a tenant?"; I don't think we care about who "leads" a tenant unless that impacts our permissions model.

Reply:
I think it does provide some MOC-specific context that may be useful for MOC staff? I don't feel super strongly either way.